source/_integrations/idasen_desk.markdown

```diff
@@ -0,0 +1,23 @@
+---
+title: IKEA IDÅSEN Desk
+description: Control and monitor your IKEA Idåsen standing desk.
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_release: '2023.10'
+ha_category:
+  - Desk
```

Comment:
Note to self: Need to check if this is a valid category but traveling at the moment so I can't until I get back to a laptop